mlir/utils/jenkins/Jenkinsfile.nightly-perf

```diff
@@ -24,17 +24,16 @@ pipeline {
                                 url: 'https://github.com/ROCmSoftwarePlatform/MIOpen.git'
                     cmake arguments: "-P install_deps.cmake --minimum",\
                                 installation: "InSearchPath"
-                    cmakeBuild generator: 'Ninja',\
-                                buildDir: 'build',\
+                    cmakeBuild buildDir: 'build',\
                                 buildType: 'Release',\
                                 installation: 'InSearchPath',\
-                                steps: [[args: "MIOpenDriver"]],\
                                 cmakeArgs: '''
                                  -DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++
                                  -DMIOPEN_BACKEND=HIP
                                  -DCMAKE_PREFIX_PATH="/opt/rocm/hip;/opt/rocm/;/usr/local" \
                                  -DCMAKE_INSTALL_PREFIX=/opt/rocm/ ..
                                 '''
+                    sh 'cd build; make -j install'
```

Comment:
Why `make -j install` instead of `make MIOpenDriver`?

Reply:
I copied it from the script you shared with me. https://github.com/ROCmSoftwarePlatform/tensorflow-benchmark/blob/main/resnet50/clone_miopen_repo.sh

Reply:
That script requires MIOpen to be installed to the system because TF dependent on an installed MIOpen. In our case, this is necessary because the script is dependent on MIOpenDriver only.

I shared that script with you just so you know about `cmake -P install_deps.cmake --minimum`, not that you should follow all details in it.